content/docs/demos/statefulsets.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "Configure Stateful Applications with OSM"
```

Comment:
"with OSM" is implicit. Let's make the titles consistent with other titles for the demo docs.
`Configure Stateful Applications`